cinema/views.py

```diff
@@ -1 +1,83 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import (
+    Genre,
+    Actor,
+    Movie,
+    MovieSession,
+    Order,
+    Ticket,
+    CinemaHall
+)
+from cinema.serializers import (
+    GenreSerializer,
+    ActorSerializer,
+    MovieSessionSerializer,
+    OrderSerializer,
+    TicketSerializer,
+    CinemaHallSerializer,
+    MovieSerializer,
+    MovieSessionListSerializer,
+    MovieListSerializer,
+    MovieSessionRetrieveSerializer,
+    MovieRetrieveSerializer
+)
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+    read_only_fields = "id"
```

Comment:
The `read_only_fields` attribute is not applicable here in the viewset. It should be defined within the `Meta` class of the serializer if needed. Consider removing it from the viewset.